CMakeLists.txt

```diff
@@ -180,6 +180,12 @@ option(ENABLE_JSON "Enable JSON" ON)
 option(ENABLE_MONGODB "Enable MongoDB" ON)
 option(ENABLE_DATA_SQLITE "Enable Data SQlite" ON)
 option(ENABLE_REDIS "Enable Redis" ON)
+option(ENABLE_DNSSD "Enable DNSSD" ON)
```

Comment:
why is this `ON`, when both Avahi and Bonjour are `OFF`? It should be off, this will be in the repository, but will not go into releases (at least not immediately).

Reply:
@aleks-f  I off options